public/app/features/explore/TableContainer.tsx

```diff
@@ -43,6 +43,13 @@ export class TableContainer extends PureComponent<TableContainerProps> {
 
     return (
       <Collapse label="Table" loading={loading} collapsible isOpen={showingTable} onToggle={this.onClickTableButton}>
+        {!tableResult?.length && (
+          <div className="logs-panel-meta">
+            <div className="logs-panel-meta__item">
+              <span className="logs-panel-meta__value">0 series returned</span>
+            </div>
+          </div>
+        )}
         {tableResult && <Table data={tableResult} width={tableWidth} height={height} onCellClick={onClickCell} />}
```

Comment:
Hi Lukasz, good job! One thing that I would change is to render either `0 series returned` or `Table`, not both if the tableReuslt.length is falsy. So maybe I would create variable `hasTableResult` and based on if it is falsy or truthy, I would render  `0 series returned` or `Table`. 
What do you think? See images as reference.
![image](https://user-images.githubusercontent.com/30407135/72718329-0a1bdf80-3b76-11ea-82ad-ff4f3357fb17.png)

![image](https://user-images.githubusercontent.com/30407135/72718377-21f36380-3b76-11ea-8416-29ace4725ee1.png)

Reply:
You're absolutely right. Fixed.